Sources/DepoCore/Services/StaticLibraryBuilderService.swift

```diff
@@ -0,0 +1,148 @@
+//
+// Copyright © 2020 Rosberry. All rights reserved.
+//
+
+import Foundation
+import PathKit
+
+public final class StaticLibraryBuilderService: ProgressObservable {
+
+    public typealias Output = String
```

Comment:
I don't see it being used, is it necessary?

Reply:
It's used in build(scheme:) func

Reply:
ok